src/locales/nl_NL/nl_NL.json

```diff
@@ -0,0 +1,447 @@
+        "list_shows_address": "Shows your <strong>current<\/strong> address",
+        "subtitle": "You’ll need a document that:",
+        "title": "Let’s verify your %{country} address"
+    },
+    "selfie_capture": {
+        "alert": {
+            "camera_inactive": {
+                "detail": "Controleer dat deze verbinding heeft en werkt. U kunt <fallback>het verificatieproces ook op uw mobiele telefoon voortzetten<\/fallback>",
+                "detail_no_fallback": "Zorg dat de camera op uw apparaat werkt",
+                "title": "Doet uw camera het niet?"
+            },
+            "camera_not_working": {
+                "detail": "Hij is mogelijk losgekoppeld. <fallback>Probeer het met uw telefoon<\/fallback>.",
+                "detail_no_fallback": "Zorg dat de camera op uw apparaat werkt",
+                "title": "Camera doet het niet"
+            },
+            "timeout": {
+                "detail": "Vergeet niet op de knop te drukken als u klaar bent. <fallback>Herhaal de stappen voor het opnemen van een filmpje<\/fallback>",
+                "title": "Het lijkt erop dat het te lang duurde"
+            }
+        },
+        "body": "Houd uw gezicht binnen de ovaal",
+        "button_accessibility": "Neem een foto",
+        "button_primary": "Start een gezichtsscan",
+        "frame_accessibility": "Waargave vanaf camera",
+        "title": "Houd uw gezicht binnen de ovaal"
+    },
+    "selfie_confirmation": {
+        "image_accessibility": "Foto van uw gezicht",
+        "subtitle": "Zorg dat uw volledige gezicht zichtbaar is",
+        "title": "Controleer uw selfie"
+    },
+    "selfie_intro": {
+        "button_primary": "Doorgaan",
+        "list_accessibility": "Tips voor een goede selfie",
+        "list_item_face_forward": "Kijk recht vooruit en zorg dat uw ogen goed te zien zijn",
+        "list_item_no_glasses": "Zet uw bril af indien nodig",
+        "subtitle": "We vergelijken deze met uw document",
+        "title": "Neem een selfie"
+    },
+    "sms_sent": {
+        "info": "Tips",
+        "info_link_expire": "Uw link verloopt over een uur",
+        "info_link_window": "Houd dit venster open als u uw mobiele telefoon gebruikt",
+        "link": "Stuur link opnieuw",
+        "subtitle": "We hebben een beveiligde link gestuurd naar %{number}",
+        "subtitle_minutes": "Het kan een paar minuten duren voordat u deze ontvangt",
+        "title": "Check uw mobiele telefoon"
+    },
+    "switch_phone": {
+        "info": "Tips",
+        "info_link_expire": "Uw mobiele link verloopt over een uur",
+        "info_link_refresh": "Ververs deze pagina niet",
+        "info_link_window": "Houd dit venster open als u uw mobiele telefoon gebruikt",
+        "link": "Annuleren",
+        "subtitle": "Zodra u klaar bent, leiden we u naar de volgende stap",
+        "title": "Verbonden met uw mobiele telefoon"
+    },
+    "upload_guide": {
+        "button_primary": "Upload foto",
```

Comment:
Is this dutch or english? 😅

Reply:
Should be Dutch...could be Malay also TBH since Malay borrows words from Dutch. 😅